src/applications/personalization/dashboard/tests/actions/claims.unit.spec.jsx

```diff
@@ -0,0 +1,164 @@
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+import {
+  mockFetch,
+  setFetchJSONFailure,
+  setFetchJSONResponse,
+} from '@department-of-veterans-affairs/platform-testing/helpers';
+
+import { createClaimsSuccess } from '../../mocks/claims';
+import { createAppealsSuccess } from '../../mocks/appeals-success';
+import { getAppealsV2, getClaims } from '../../actions/claims';
+
+import {
+  FETCH_CLAIMS_PENDING,
+  FETCH_CLAIMS_ERROR,
+  FETCH_CLAIMS_SUCCESS,
+} from '../../utils/claims-helpers';
+import {
+  FETCH_APPEALS_PENDING,
+  FETCH_APPEALS_SUCCESS,
+  USER_FORBIDDEN_ERROR,
+  RECORD_NOT_FOUND_ERROR,
+  VALIDATION_ERROR,
+  BACKEND_SERVICE_ERROR,
+  FETCH_APPEALS_ERROR,
+} from '../../utils/appeals-helpers';
+
+describe('/actions/claims', () => {
+  describe('getAppealsV2', () => {
+    let dispatchSpy;
+    let oldDataLayer;
+    beforeEach(() => {
+      mockFetch();
```

Comment:
So I have heard whisperings that we may move to using Mock Service Worker within unit tests instead of mocking the global fetch via `mockFetch` since I guess the global mocking has introduced some flakiness / memory leak issues potentially.

That being said, this is still the 'official' way of mocking fetch calls, so I say we roll with it for now, and if we continue to refactor tests if/when MSW is recommended then we will cross that bridge at that time.